tgarchive/example/config.yaml

```diff
@@ -14,9 +14,17 @@ download_avatars: True
 avatar_size: [64, 64] # Width, Height.
 media_dir: "media"
 
+# Takeout mode allows you to fetch messages at a higher rate than the standard mode.
+# It is the method used in the desktop client to export data.
+# You can use a larger fetch_batch_size. Set this as False to use the standard mode.
+use_takeout: True
+
 # These should be configured carefully to not get rate limited by Telegram.
 # Number of messages to fetch in one batch.
-fetch_batch_size: 2000
+# For takeout mode (use_takeout: True)
+fetch_batch_size: 4000
```

Comment:
Since non-takeout mode is the default option, this should remain at `2000`. The `use_takeout` comments can explain tha this number can be increased.

Reply:
Sure!!